packages/tasks/src/local-apps.ts

```diff
@@ -0,0 +1,119 @@
+import type { ModelData } from "./model-data";
+import type { PipelineType } from "./pipelines";
+
+/**
+ * Elements configurable by a local app.
+ */
+export type LocalApp = {
+	/**
+	 * Name that appears in buttons
+	 */
+	prettyLabel: string;
+	/**
+	 * Link to get more info about a local app (website etc)
+	 */
+	docsUrl: string;
+	/**
+	 * main category of app
+	 */
+	mainTask: PipelineType;
```

Comment:
Should we have a list here? I don't think we want a 1:1 mapping of local app to pipeline, e.g. if lmstudio supports tti later on, we should just add one to the list. Similar to https://github.com/huggingface/huggingface.js/blob/main/packages/tasks/src/library-to-tasks.ts#L29

Reply:
this is only to categorize in the listing UI, though. i.E. it doesn't control on which page it will be displayed

Reply:
marking this as resolved